content/en/security/cloud_siem/content_packs.md

```diff
@@ -135,6 +152,40 @@ The [Cloudflare Content Pack][25] includes:
 - An interactive dashboard
 - Workflow Automation
 
+### Cisco Meraki
+
+Monitor Cisco Meraki logs and identify attacker activity.
+
+The [Cisco Meraki Content Pack][35] includes:
+- [Detection Rules][36]
+- An interactive dashboard
+
+### Palo Alto Networks Firewall
+
+Analyze traffic and detect threats with Palo Alto Networks Firewall.
+
+The [Palo Alto Networks Firewall Content Pack][37] includes:
+- [Detection Rules][38]
+- An interactive dashboard
+
+### Cisco Umbrella
+
+Collect and monitor logs from Cisco Umbrella to gain insights into DNS and Proxy logs
+
+The [Cisco Umbrella Content Pack][39] includes:
+- [Detection Rules][40]
+- An interactive dashboard
+
+## Web Security Content Packs
+
+### nginx
```

Comment:
```suggestion
### Nginx
```

Reply:
The previous update had also changed the capitalization. Which way do we want it? `Nginx`, `nginx`, or `NGINX`?

Reply:
Let's go with `NGINX`